fastapi_error_logger/__init__.py,sha256=0WbPfjUBkdsW2sB8GvOpsKaGoqNS-s__BvTjmGHXutg,24
fastapi_error_logger/api_logger.py,sha256=2ra-zgkFGHcWKCg9uzwuttnwMpBgHkqQeDySCedQc5E,3044
fastapi_error_logger/exceptions.py,sha256=UOzA3qwbjCKHqnUKq1qv2pkyXnuhlfO9vXQ1G28Qe8o,3072
fastapi_error_logger/models.py,sha256=TX1mURPPBJc6bQ7StIH4r3YcrxTPK_yIjyoB7SrhYCg,1631
fastapi_error_logger-0.1.2.dist-info/LICENSE,sha256=zpkxbuRzzmU0GH6RrOOAtfrDElXRLE624dLi2sYd9J8,1092
fastapi_error_logger-0.1.2.dist-info/METADATA,sha256=jEhPRyF63y7sPtt0C78zkRFDLmPxJMQViUXNmrKSqs4,1838
fastapi_error_logger-0.1.2.dist-info/WHEEL,sha256=Xo9-1PvkuimrydujYJAjF7pCkriuXBpUPEjma1nZyJ0,92
fastapi_error_logger-0.1.2.dist-info/top_level.txt,sha256=9UvEudfJ0qPs0SODwoDdrP1pWzysbKqfqjdDJtVig4s,21
fastapi_error_logger-0.1.2.dist-info/RECORD,,
